.github/fabricbot.json

```diff
@@ -271,10 +201,9 @@
             }
           ]
         },
-        "eventType": "issue",
+        "eventType": "pull_request",
```

Comment:
This task was previously applying only to issues, but it was intended to apply to pull requests. This corrects the current configuration and also prepares it to port properly.